lib/core/iam/ecs.json

```diff
@@ -191,6 +191,11 @@
           ]
         }
       }
+    },
+    {
+      "Effect": "Allow",
+      "Action": "secretsmanager:GetSecretValue",
+      "Resource": "*"
```

Comment:
this seems overly permissive. i realize this is a json file, but can we scope somehow?

Reply:
Not sure how to fix this due to a circular dependency. Serve has a dependency on the IAM stack, any suggestions?